install/lib/class.FroxlorInstall.php

```diff
@@ -504,6 +505,15 @@ private function _doSettings(&$db)
 			$this->_updateSetting($upd_stmt, 'error', 'system', 'errorlog_level');
 		}
 
+		if (file_exists(dirname(__DIR__).'/../lib/configfiles/'.$this->_data['distribution'].'.xml')) {
```

Comment:
Why not do it the way we do it in admin_configfiles? See https://github.com/Froxlor/Froxlor/blob/master/admin_configfiles.php#L98

Reply:
will do

Reply:
Waiting on this